parm/config/gefs/config.base

```diff
@@ -81,6 +81,10 @@ export CHGRP_CMD="@CHGRP_CMD@"
 export NCDUMP="${NETCDF:-${netcdf_c_ROOT:-}}/bin/ncdump"
 export NCLEN="${HOMEgfs}/ush/getncdimlen"
 
+# Stop gap fix for wgrib with spack-stack 1.6.0
+# TODO Remove this when spack-stack issue #1097 is resolved
+export WGRIB="${WGRIB:-${grib_util_ROOT:-}/wgrib}"
```

Comment:
The module files are already defining `WGRIB`, why is it needed here?

Reply:
The grib-util module file is defining `WGRIB2` by accident.  See line 45 of `/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/modulefiles/intel/2021.5.0/grib-util/1.3.0.lua`:
```lua
setenv("WGRIB2", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev-rocky8/install/intel/2021.5.0/grib-util-1.3.0-ms2iw3h/bin/wgrib2")
```
The spack-stack developers are working on fixing this in the code and then will update the module files locally.  Until then, this is needed for a couple of the wave scripts.

Reply:
_Our_ modulefiles are defining WGRIB2 because the spack-stack ones weren't. See https://github.com/NOAA-EMC/global-workflow/blob/develop/modulefiles/module_base.hera.lua#L43 or any of the other spack-based module_base files.

Reply:
Should this be in ush/load_ufswm_modules.sh instead? We already have some updates there for WGRIB2

Reply:
@WalterKolczynski-NOAA Agreed, our module file is doing that for `WGRIB2`, but not `WGRIB`.  I think moving this line over to the `module_base`s would be fine.

Reply:
OH! Sorry, I was misreading it. Yeah, it should be in the module file.

Reply:
Moved.